optuna/terminator/improvement/gp/botorch.py

```diff
@@ -84,7 +84,7 @@ def predict_mean_std(
         return mean.detach().numpy(), std.detach().numpy()
 
 
-def _convert_trials_to_tensors(trials: List[FrozenTrial]) -> Tuple[torch.Tensor, torch.Tensor]:
+def _convert_trials_to_tensors(trials: List[FrozenTrial]) -> Tuple["torch.Tensor", "torch.Tensor"]:
```

Comment:
Does it sound reasonable to use `from __future__ import annotations` instead of making it string? Then we can also replace `typing.List` and `typing.Tuple` with `list` and `tuple`.

```suggestion
def _convert_trials_to_tensors(trials: list[FrozenTrial]) -> tuple[torch.Tensor, torch.Tensor]:
```

Reply:
Sounds good, let me adopt it!

Reply:
PTAL.